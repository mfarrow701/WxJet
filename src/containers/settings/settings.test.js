import React from 'react';
import {shallow} from 'enzyme';
import Settings from './settings';

describe('Settings container', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Settings/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the settings container after load', () => {
        expect(component).toMatchSnapshot();
        expect(component.find('.Settings h1').text()).toEqual('Settings');
    });
});
