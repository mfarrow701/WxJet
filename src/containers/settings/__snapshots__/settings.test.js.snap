// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings container should render the settings container after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Settings />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Settings
</h1>,
      "className": "Settings",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Settings",
      },
      "ref": null,
      "rendered": "Settings",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Settings
</h1>,
        "className": "Settings",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Settings",
        },
        "ref": null,
        "rendered": "Settings",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
